crates/ruff_dev/src/format_dev.rs

```diff
@@ -239,78 +261,88 @@ fn format_dev_multi_project(args: &Args) -> anyhow::Result<bool> {
         None => None,
     };
 
-    #[allow(clippy::print_stdout)]
     for project_path in project_paths {
-        bar.suspend(|| println!("Starting {}", project_path.display()));
```

Comment:
a nice side effect is that tracing-indicatif makes sure the progress bars and the logging output don't collide